README.md

```diff
@@ -193,6 +197,12 @@ Notes:
 | [Toggle Location Services](http://appium.io/docs/en/commands/device/network/toggle-location-services/) |
 | [Unlock](http://appium.io/docs/en/commands/device/interactions/unlock/) |
 
+### Mobile Commands
+
+| Command	                 |   Description                                                             |	Argument       |	Argument Example           |
+|----------------------------|---------------------------------------------------------------------------|-----------------|-------------------------------|
+| mobile:pressButton         |   Press a physical button. The available button options can be found [here](https://github.com/YOU-i-Labs/appium-youiengine-driver/blob/master/doc/mobileCommands.md#supported-buttons) |	{name}         |	{name: "Gamepad0"}             |
+
```

Comment:
I'm wondering if we should have a complete example of a command to make it easier to figure out.

Reply:
Is the example not here?
http://appium.io/docs/en/commands/mobile-command/

Reply:
added a link to it